Makefile.am

```diff
@@ -73,6 +82,9 @@ CLEANFILES = \
 	$(resource_DATA) \
 	$(service_DATA) \
 	src/com.endlessm.DiscoveryFeed \
+	$(gsettings_SCHEMAS) \
+        data/gschemas.compiled \
+        data/com.endlessm.DiscoveryFeed.gschema.valid \
```

Comment:
I think these two lines can be removed from here

Reply:
(if they need to be kept for whatever reason, s/spaces/tabs)

Reply:
So "data/gschemas.compiled" does not get created anymore, and "data/com.endlessm.DiscoveryFeed.gschema.valid" gets automatically cleaned up.